external.bzl

```diff
@@ -333,6 +333,14 @@ def _go_dependencies():
         importpath = "github.com/sourcegraph/jsonrpc2",
     )
 
+    maybe(
+        go_repository,
+        name = "com_github_hanwen_go_fuse",
```

Comment:
Tangentially: we're clearly not shy about adding additional external dependencies, but I wonder if this and the language server can't be reasonably pulled into their own repo.